src/capabilities/capabilities.ts

```diff
@@ -32,6 +32,11 @@ export enum Capability {
   SAVE_SNIPPETS = "save_snippets",
   BETA_CAPABILITY = "beta",
   FIRST_SUGGESTION_DECORATION = "first_suggestion_hint_enabled",
+  DEBOUNCE_VALUE_300 = "DEBOUNCE_VALUE_300",
```

Comment:
I'm not sure if this is converted to lowercase somewhere, but this is how it should be:
![image](https://user-images.githubusercontent.com/10290661/209675511-20a78902-3f4c-4bf3-aa7d-ac04e1f3b5a2.png)